src/main/java/bio/terra/cli/command/notebook/Launch.java

```diff
@@ -0,0 +1,66 @@
+package bio.terra.cli.command.notebook;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Workspace;
+import bio.terra.cli.businessobject.resource.AwsSageMakerNotebook;
+import bio.terra.cli.command.shared.BaseCommand;
+import bio.terra.cli.command.shared.options.Format;
+import bio.terra.cli.command.shared.options.NotebookInstance;
+import bio.terra.cli.command.shared.options.WorkspaceOverride;
+import bio.terra.cli.exception.UserActionableException;
+import bio.terra.cli.service.WorkspaceManagerServiceAws;
+import bio.terra.workspace.model.CloudPlatform;
+import java.net.URL;
+import org.json.JSONObject;
+import picocli.CommandLine;
+
+/** This class corresponds to the third-level "terra notebook launch" command. */
+@CommandLine.Command(
+    name = "launch",
+    description = "Launch a running Notebook instance within your workspace.",
+    showDefaultValues = true)
+public class Launch extends BaseCommand {
+  private static final String PROXY_URL = "proxy_url";
+  @CommandLine.Mixin NotebookInstance instanceOption;
+  @CommandLine.Mixin WorkspaceOverride workspaceOption;
+  @CommandLine.Mixin Format formatOption;
+
+  @CommandLine.Option(
+      names = "--proxy-view",
+      description = "AWS Sagemaker Notebook view to be launched: ${COMPLETION-CANDIDATES}.",
+      defaultValue = "JUPYTER",
```

Comment:
I think the lab view is probably the better default.

Reply:
Done